.github/workflows/ci.yml

```diff
@@ -189,6 +189,7 @@ jobs:
       - uses: ./.github/actions/upload
         with:
           test-report-retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
+          upload-heap-dump: "true"
```

Comment:
make conditional?

this job has no secrets, but what if someone adds secrets later?
they may easily miss the fact we upload heap dumps